### PR TITLE
Bugfix/3273 error check wazuh no security

### LINKED ISSUE #3273: Error api/check-wazuh in no security environment
| Wazuh | Elastic |
| ----- | ------- | 
| 4.2   | 7.10.2     |

**Description**
`api/check-wazuh`eror in No security environment.
Screen Shot 2021-05-19 at 16 49 35

- `server/controllers/wazuh-api.ts > isWazuhDisabled` launch error and return wazuh is `disabled` if user don't have roles,

### PR DESCRIPTION
**Pull request by Machi3mfl**
Hi Team,
This PR resolves:
- Error api/check-wazuh in no security environment
Close #3273

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,8 +31,9 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fix typo error in Settings/Configuration [#3234](https://github.com/wazuh/wazuh-kibana-app/pull/3234)
 - Fixed fields overlap in the agent summary screen [#3217](https://github.com/wazuh/wazuh-kibana-app/pull/3217)
 - Fixed the `Visualize` button is not displaying when expanding a field in the Events sidebar [#3237](https://github.com/wazuh/wazuh-kibana-app/pull/3237)
-- Add error when add sample data fails [#3241] (https://github.com/wazuh/wazuh-kibana-app/pull/3241)
-- Fix modules are missing in the agent menu [#3244] (https://github.com/wazuh/wazuh-kibana-app/pull/3244)
+- Add error when add sample data fails [#3241](https://github.com/wazuh/wazuh-kibana-app/pull/3241)
+- Fix modules are missing in the agent menu [#3244](https://github.com/wazuh/wazuh-kibana-app/pull/3244)
+- Fixed error api/check-wazuh in no security environment [#3292](https://github.com/wazuh/wazuh-kibana-app/pull/3292)
```
Comment (Desvelao) [inline CHANGELOG.md:36]: This PR fixes a feature that releases in the same revision. I think we maybe should add the fix or move the PR to the feature instead.

**Review by Desvelao — CHANGES_REQUESTED** [review]
Code Review: ❌ Remove or move the change in the changelog. It is due to the feature is new and is not released still.
Test: ✅

**Review by pablomarga — APPROVED** [review]
Test: :heavy_check_mark: 
CR: :heavy_check_mark: 
LGTM!

**Review by Desvelao — APPROVED** [review]
***Changes***
- Moved PR number to a the change with the feature

**Pull request merged**